src/components/sidebar/custom/Custom.js

```diff
@@ -30,13 +36,48 @@ export default function Custom() {
     friday: false,
   });
 
+  const handleTextFieldChange = changeFunc => (event) => { changeFunc(event.target.value); };
   const handleChange = name => (event) => {
     setState({ ...state, [name]: event.target.checked });
   };
 
   const { monday, tuesday, wednesday, thursday, friday } = state;
   const error = [monday, tuesday, wednesday, thursday, friday].filter(v => v).length < 1;
 
+  const dowMap = ['Mo', 'Tue', 'Wed', 'Thurs', 'Fri'];
+  const dispatch = useDispatch();
+  const handleSubmit = () => {
+    const dow = Object.values(state).map((v, i) => v ? dowMap[i] : null).filter(v => v);
+    const [startHour, startMinute] = startTime.split(':').map(x => parseInt(x, 10));
+    const [endHour, endMinute] = endTime.split(':').map(x => parseInt(x, 10));
+    const schedule = {
+      dow,
+      start: {
+        hour: startHour,
+        minute: startMinute,
+      },
+      end: {
+        hour: endHour,
+        minute: endMinute,
+      },
+      location: '',
+    };
+    const section = {
+      id: (Math.random() * 100).toString(),
```

Comment:
Maybe in the future we'll replace this with `uuid` but as of right now this is probably fine.